src/app/signup/signupComponent.tsx

```diff
@@ -45,20 +51,6 @@ export default function SignUp() {
   const [passwordError, setPasswordError] = useState(false);
   const [emailError, setEmailError] = useState(false);
 
-  useEffect(() => {
```

Comment:
why was this removed? Shawn just implemented it.

Reply:
My bad, I think it got removed when I merged?

Reply:
Will add it back